CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ Join us on IRC at **#awesome-go** on freenode [web access](http://webchat.freeno
 - **To add to the list:** Submit a pull request
 - **To remove from the list:** Open an issue
 
-To set this list apart from and compliment the excellent [Go wiki Projects page](https://code.google.com/p/go-wiki/wiki/Projects), awesome-go is a specially curated list for high-quality, actively maintained Go packages and resources.
+To set this list apart from and compliment the excellent [Go wiki Projects page](https://github.com/golang/go/wiki/Projects), awesome-go is a specially curated list for high-quality, actively maintained Go packages and resources.
```

Comment:
Maybe this should be https://golang.org/wiki/Projects instead?

If you go to https://code.google.com/p/go-wiki/wiki/Projects, it says at the top:

> This page has moved: https://golang.org/wiki/Projects

However, if you go to https://golang.org/wiki/Projects, it currently redirects you (via 302 found) https://github.com/golang/go/wiki/Projects. However, just in case that redirect changes in the future, maybe it makes more sense to use the "canonical" url https://golang.org/wiki/Projects?